libnopegl/src/backends/vk/gpu_ctx_vk.h

```diff
@@ -58,8 +58,8 @@ struct gpu_ctx_vk {
     int width;
     int height;
 
-    int nb_in_flight_frames;
-    int cur_frame_index;
+    size_t nb_in_flight_frames;
+    size_t cur_frame_index;
```

Comment:
Same question here (size_t vs uint32_t).

Reply:
Fine with me: do you wish to use `for (u32 i = ...` and keep the sign extend then?

Reply:
I'm in favor in consistency, so you can go with `for (u32 i = ...`. Thanks for the explanation.

Reply:
Done